src/gtimelog/main.py

```diff
@@ -63,15 +63,36 @@ def mark_time(what=None):
     SHORTCUTS_UI_FILE,
     UI_FILE,
 )
-from .utils import require_version
+from .utils import (
+    internationalized_format_duration,
```

Comment:
That's a mouthful, how about `format_duration_i18n`?  Or `translated_duration()`?